sorbet/config

```diff
@@ -1,6 +1,7 @@
 --dir=.
 --ignore=tmp/
 --ignore=vendor/
+--ignore=.bundle/
```

Comment:
What makes this change necessary?

Reply:
Oh, nothing, this is just because I run sorbet locally, and since I had files at `bundler/helpers/v1/.bundle` and `bundler/helpers/v2/.bundle` (probably because of having run bundler specs), sorbet showed a bunch of failures in those installed gems.